.circleci/config.yml

```diff
@@ -51,18 +53,19 @@ jobs:
           name: Python Tests
           command: docker-compose run web python-tests
 
-      - run:
-          name: Start Django/Gunicorn server
-          command: docker-compose up web
-          background: true
-
       - run:
           name: Contract tests
-          command: docker-compose run web contracttest
+          command: |
+            docker-compose up -d web
+            docker-compose run web contracttest
```

Comment:
I thought about making a separate Docker Compose for `contracttest` as well, so that this could have the exact same shape as the JS tests, but I'm not sure that's worth it.